src/main/kotlin/com/sourcegraph/cody/chat/ui/LlmDropdown.kt

```diff
@@ -44,9 +43,9 @@ class LlmDropdown(
     data.models.forEach(::addItem)
     data.models.find { it.default }?.let { this.selectedItem = it }
 
-    val activeAccountType = CodyAuthenticationManager.instance.getActiveAccount(project)
-    isEnabled =
-        !didSendFirstMessage && !(activeAccountType.isEnterpriseAccount() || model.size <= 1)
+    val isEnterpriseAccount =
+        CodyAuthenticationManager.instance.getActiveAccount(project)?.isEnterpriseAccount() ?: false
+    isEnabled = !didSendFirstMessage && !isEnterpriseAccount && model.size > 1
```

Comment:
hah - it looks much better now 😅 

<img width="547" alt="image" src="https://github.com/sourcegraph/jetbrains/assets/19799111/225f6659-55fe-40c9-a73e-6a5b294054e0">